[FR/EN] Add `Mapping Techniques/Basics`, `Formations`, `Jumps`, `Making Good Sliders`, improvement of EN files (#5655)
ppy/osu-wiki: wiki/Mapping_Techniques/Formations/en.md

> <!-- TODO: need to be rewrite, the article being very old -->

Author: ```suggestion
<!-- TODO: Needs to be rewritten, the article does not match with other articles (writing style, formatting).
Also check if the article matches with ASC. See #5729 for more informations -->
```

### Blanket Combos

*Also known as: **Blanket Formation***

Nominator: [m980](https://osu.ppy.sh/users/3288)

> **Blanket combos** is made up of a slider or a hit circle (as the base) with a slider that surrounds that object.

Reviewer: ```suggestion
**Blanket combos** are made up of a slider or a hit circle (as the base) with a slider that surrounds that object.
```

Nominator: [mm201](https://osu.ppy.sh/users/30655)

> **Honeycomb beats** is a stream of hit circles that are usually half beats apart.

Reviewer 2: are streams* ? You use plural for some stuff so i don't know

Reviewer: ```suggestion
**Honeycomb beats** are streams of hit circles that are usually half beats apart.
```

Nominator: [Daru](https://osu.ppy.sh/users/32480)

**The kickstream** uses a kick slider (a really short slider) inside of a stream.
This forces the player to pause in the middle of the stream to complete the kick, before continuing the stream.

### Tornado Combos

Nominator: [Dragvon](https://osu.ppy.sh/users/126927)

> **Tornado formation** are sets of notes that forces the player to tap circles in a circular motion at a high rate.

Reviewer: ```suggestion
**Tornado formations** are sets of notes that forces the player to tap circles in a circular motion at a high rate.
```

Nominator: QiiXshinez

> The **diamond formation** is just like the [square formation](#square-formation) or [square formation v2](#square-formation-v2) but uses rhombi instead of squares.

Reviewer 2: Diamond formation* ? Should have a uppercase letter

### Double Stacks

Nominators: [-Xero-](https://osu.ppy.sh/users/179314) and [F1r3tar](https://osu.ppy.sh/users/5590317)

> **Double stacks** uses two notes (or one note and a slider end) that are stacked on top of each other.

Reviewer: ```suggestion
**Double stacks** use two notes (or one note and a slider end) that are stacked on top of each other.
```

### QQ Diamonds

Nominator: [Lybydose](https://osu.ppy.sh/users/64501)

**QQ diamonds** are four circles placed one beat apart from each other in a diamond pattern along the four edges (not corners) of the screen.
This is similar to [diamond formation](#diamond-formation), but these are placed at the ends of the x-axis and y-axis.

In gaming, "QQ" originates from Warcraft II, where the player is able to press `Alt` + `Q` + `Q` to immediately leave the battle.
When a player in the chat tells you to "QQ", they are telling you to leave because you are unskilled to play.

### Reversed Zig-zag Sliders

Nominator: *none*

**Reversed zig-zag sliders** is a variation of the [zig-zag Sliders](#zig-zag-sliders) but has the sliders point outwards and away from the next slider.
This creates a quick jump for the player.

### Square Formation

Nominator: Beuchi-chan

> **Square formation** is 4 hit circles forming a square shape.

Reviewer: ```suggestion
**Square formations** consists of 4 hit circles forming a square shape.
```

### Double Stacks

Nominators: [-Xero-](https://osu.ppy.sh/users/179314) and [F1r3tar](https://osu.ppy.sh/users/5590317)

**Double stacks** use two notes (or one note and a slider end) that are stacked on top of each other.

### QQ Diamonds

Nominator: [Lybydose](https://osu.ppy.sh/users/64501)

**QQ diamonds** are four circles placed one beat apart from each other in a diamond pattern along the four edges (not corners) of the screen.
This is similar to [diamond formation](#diamond-formation), but these are placed at the ends of the x-axis and y-axis.

In gaming, "QQ" originates from Warcraft II, where the player is able to press `Alt` + `Q` + `Q` to immediately leave the battle.
When a player in the chat tells you to "QQ", they are telling you to leave because you are unskilled to play.

### Reversed Zig-zag Sliders

Nominator: *none*

**Reversed zig-zag sliders** is a variation of the [zig-zag Sliders](#zig-zag-sliders) but has the sliders point outwards and away from the next slider.
This creates a quick jump for the player.

### Square Formation

Nominator: Beuchi-chan

> **Square formations** consists of 4 hit circles forming a square shape.

Reviewer 3: ```suggestion
**Square formations** consist of 4 hit circles forming a square shape.
```

Nominators: Beuchi-chan and [mm201](https://osu.ppy.sh/users/30655)

**Square formation v2** are pretty much the same as [square formation](#square-formation), but the order of the circles forces you to go diagonally through the square.

### Triangle Beats

*Also known as: **Triangle Formation***

Nominator: [kingcobra52](https://osu.ppy.sh/users/9934)

> **Triangle beats** is a very common mapping technique where 3 hit circles are arranged in a triangle.

Reviewer 2: For this one, I'm not sure if we should but "are" insted of "is", but I'm commenting to be sure

Reviewer: can be left as is, because "triangle beats" is treated a single term when referred to as one "mapping technique"

Nominators: Beuchi-chan and [mm201](https://osu.ppy.sh/users/30655)

> **Square formation v2** are pretty much the same as [square formation](#square-formation), but the order of the circles forces you to go diagonally through the square.

Reviewer 3: ```suggestion
**Square formation v2** is pretty much the same as [square formation](#square-formation), but the order of the circles forces you to go diagonally through the square.
```
